src/ui/components/atoms/label-share/styles.scss

```diff
@@ -0,0 +1,54 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.label-share {
+  background: $ads-primary-main;
+  background-color: $ads-text-main; 
+  height: 2rem; 
+  padding: 1rem;     
+  width: 40rem;
+
+  @include respond-to('small') {
+    font-size: 1rem;
+    width: 7.5rem;
+  }
+
+  @include respond-to('medium') {
+    font-size: 1rem;
+    width: 7.5rem;
+  }
+
+  @include respond-to('large') {
+    font-size: 2rem;
+    width: 16rem;
+  }
+
+  @include respond-to('extra-large') {
+    font-size: 2rem;
+    width: 16rem;
+  }
+}
+
+.label-share__text {    
+  font-family: Lato, sans-serif;
+  font-size: 1rem;
+  font-weight: $ads-font-weight-bold;
+  letter-spacing: 0.0625rem;
+  line-height: 1;
+  padding: $ads-letter-spacing;
```

Comment:
I'm not sure to use the variable $ads-letter-spacing like padding. why are you using it like that??